pkg/aws/aws.go

```diff
@@ -22,6 +27,65 @@ import (
 	"github.com/pkg/errors"
 )
 
+const tagKeyHostname = "devpod:hostname"
+
+type Machine struct {
```

Comment:
nit: would you mind pulling this and the route53 changes out into separate files in the same package? It's getting a bit crowded in here

Reply:
done